src/functionalTest/resources/application-functional.yaml

```diff
@@ -9,4 +9,4 @@ oauth2.client.secret: ${CA_REF_OAUTH2_CLIENT_SECRET:}
 idam.auth.clientId: ${CA_REF_OAUTH2_CLIENT_ID:}
 idam.auth.redirectUrl: ${OAUTH2_REDIRECT_URI:https://rd-caseworker-ref-api-aat.service.core-compute-aat.internal/oauth2redirect}
 s2s-secret: ${S2S_SECRET:}
-userProfUrl: http://rd-user-profile-api-aat.service.core-compute-aat.internal
\ No newline at end of file
+userProfUrl: http://rd-user-profile-api-pr-664.service.core-compute-preview.internal
```

Comment:
should not this be pointed to aat as QA is done?

Reply:
Will change this once UP PR is merged

Reply:
Changes have been made here: https://github.com/hmcts/rd-caseworker-ref-api/pull/291/commits/a7946e2fc84d69cefdf576170b18ab77b433bba4